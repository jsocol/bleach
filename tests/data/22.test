<b href="javascript:alert(1)" title="javascript:alert(2)"></b>
