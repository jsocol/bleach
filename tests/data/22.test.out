<b title="javascript:alert(2)"></b>
